src/KiteConnect.php

```diff
@@ -9,6 +9,7 @@
 use DateTimeZone;
 use Exception;
 use GuzzleHttp\Client;
```

Comment:
Do we still need this?

Reply:
Yes, Missed it. will change it.
Thanks ! 👍🏼